base/expr.jl

```diff
@@ -926,18 +926,18 @@ This can be used to limit the number of compiler-generated specializations durin
 
 # Examples
 
-```julia
+```jldoctest; setup = :(using InteractiveUtils)
 julia> f(A::AbstractArray) = g(A)
 f (generic function with 1 method)
 
-julia> @noinline Base.@nospecializeinfer g(@nospecialize(A::AbstractArray)) = A[1]
+julia> @noinline g(@nospecialize(A::AbstractArray)) = A[1]
```

Comment:
It seems a bit odd to remove the `Base.@nospecializeinfer` in the docstring for it, or?

Reply:
oops, stupid copy-paste error